src/enum_net.cpp

```diff
@@ -174,39 +174,37 @@ namespace {
 	}
 
 #if TORRENT_USE_GETIPFORWARDTABLE || TORRENT_USE_NETLINK
-	address build_netmask(int bits, int family)
+	address build_netmask(int bits, int const family)
 	{
 		if (family == AF_INET)
 		{
-			using bytes_t = boost::asio::ip::address_v4::bytes_type;
-			bytes_t b;
-			std::memset(&b[0], 0xff, b.size());
-			for (int i = int(b.size()) - 1; i > 0; --i)
+			address_v4::bytes_type b;
+			b.fill(0xff);
+			for (int i = int(b.size()) - 1; i >= 0; --i)
```

Comment:
@ssiloti do it looks good now? (with `i >= 0`)

Reply:
lgtm